package.json

```
@@ -13,102 +13,53 @@
     "packages/*"
   ],
   "engines": {
-    "node": "12.x"
+    "node": ">=12"
```

How does this, fit in with CI? https://github.com/mdx-js/mdx/blob/9443e235679f7d37cdfa436ef1b167e4bb68615d/.github/workflows/ci.yml#L14

Would it make sense to use a version union?

```suggestion
    "node": "^10 || ^12 || ^14"
```

---

I don’t think it does anything because it’s ignored: https://github.com/mdx-js/mdx/blob/e194fc1a61715549be10d56055ea4ad25533fcb1/.yarnrc#L1. And it’s in the main package.json which isn’t published.

I thought unlocking it at least would make more sense that locking to an old version, but would like to revisit it later (IMO remove it)